src/bbmap_bbsplit/config.vsh.yaml

```diff
@@ -30,7 +30,7 @@ argument_groups:
     type: boolean_true
     description: If set, only builds the index. Otherwise, mapping is performed.
   - name: "--build"
```

Comment:
What does --build do? The current description does not seem to reflect that --build should be a file